client/src/components/History/History.jsx

```diff
@@ -52,6 +52,13 @@ const HistorySidebar = () => {
       hidden: false,
       children: <HistoryBadge unread={unreadJobs} />,
     },
+    {
+      to: `${root}/jobsv2`,
```

Comment:
Should we add a "TODO" to easily find these "v2" references when we're ready to remove this feature in the future?

Reply:
[TV3-121 - Remove V2 historical jobs](https://jira.tacc.utexas.edu/browse/TV3-121)